src/Lumi/Components/Modal.purs

```diff
@@ -466,3 +488,6 @@ styles = jss
   }
   where
     alpha a = toHSLA >>> \{ h, s, l } -> hsla h s l a
+
+eventKey :: Event -> Maybe String
+eventKey = toMaybe <<< _.key <<< unsafeCoerce
```

Comment:
I don't think you need `unsafeCoerce` here, as far as I can tell [`fromEvent`](https://pursuit.purescript.org/packages/purescript-web-uievents/2.0.0/docs/Web.UIEvent.KeyboardEvent#v:fromEvent) and [`key`](https://pursuit.purescript.org/packages/purescript-web-uievents/2.0.0/docs/Web.UIEvent.KeyboardEvent#v:key) should be sufficient here.

Reply:
I think that `key` is from a package we don't have installed. We could add it, though. I don't mind either way.

Reply:
Ah, fair enough then. Up to you @kimispencer

Reply:
👍 Just added that package and updated this.